core/src/main/scala/org/bykn/bosatsu/Evaluation.scala

```diff
@@ -239,7 +248,7 @@ case class Evaluation(pm: PackageMap.Inferred, externals: Externals) {
   def evaluateLast(p: PackageName): Option[(Eval[Value], Type)] =
     for {
       pack <- pm.toMap.get(p)
-      (_, expr) <- pack.program.lets.lastOption
+      (_, rec, expr) <- pack.program.lets.lastOption
```

Comment:
ignoring the rec here. Probably isn't a problem since we can only have recursive functions and we don't evaluate main modules which have fn type, but still, we are ignoring it, and that should be fixed.